openfecli/commands/generate_partial_charges.py

```diff
@@ -0,0 +1,91 @@
+import click
+from openfecli import OFECommandPlugin
+from openfecli.parameters import MOL_DIR, YAML_OPTIONS, OUTPUT_FILE_AND_EXT, NCORES, OVERWRITE
+
+
+@click.command(
+    "charge-molecules",
+    short_help="Generate partial charges for a set of molecules."
+)
+@MOL_DIR.parameter(
+    required=True, help=MOL_DIR.kwargs["help"] + " Any number of sdf paths."
+)
+@YAML_OPTIONS.parameter(
+    multiple=False, required=False, default=None,
+    help=YAML_OPTIONS.kwargs["help"],
+)
+@OUTPUT_FILE_AND_EXT.parameter(
+    help="The name of the SDF file the charged ligands should be written to."
```

Comment:
(apologies if it's somewhere and I've missed it), We probably need a check that we're not allowing users to overwrite their input SDF? 100% can see someone accidentally passing the same SDF as arguments (or some other existing one), by accident.

Reply:
Good point! added a check and a test!